src/main/res/values/strings.xml

```diff
@@ -267,6 +267,10 @@
     <string name="select_chat">Select Chat</string>
     <string name="select_more">Select more</string>
     <string name="menu_edit_name">Edit Name</string>
+    <!-- The placeholder will be replaced by the name the contact gave themself (if any) or by an e-mail address. -->
+    <string name="edit_name_explain">Set a local nickname to replace \"%1$s\". Leave empty to use the contact\'s chosen name.</string>
```

Comment:
I think saying just "local" is not necessarily straight forward the implications for most marginal users, so emphasizing "only you will see this nickname" or something like that in some part of the text would be nice

Reply:
so maybe the first attempt was better:

"Set a nickname that will be displayed for you instead of \"%1$s\". Leave empty to use the user\'s chosen name."

Reply:
"Set a nickname that only you will see instead of "%1$s". Leave empty to use the user's chosen name."

or

"Set a nickname that will be shown to you instead of "%1$s". Leave empty to use the user's chosen name."

would be my picks but that one is fine as well, the reason I don't like it too much is that "that will be displayed for you" could be misunderstood as if the nick is for you / your nick and not the other person's

Reply:
further feedback:

> "Set a nickname that only you will see instead of "%1$s". Leave empty to use the user's chosen name."
> 
> or
> 
> "Set a nickname that will be shown to you instead of "%1$s". Leave empty to use the user's chosen name."
>
> would be my picks but that one is fine as well, the reason I don't like it too much is that "that will be displayed for you" could be misunderstood as if the nick is for you / your nick and not the other person's

i went for 

> "Set a nickname that will be shown to you instead of "%1$s". Leave empty to use the contact's chosen name."